src/Nancy/ErrorHandling/DefaultStatusCodeHandlerResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Nancy.ErrorHandling
+{
+    /// <summary>
+    /// Default error handler result
+    /// </summary>
+    public class DefaultStatusCodeHandlerResult
+    {
+        /// <summary>
+        /// Gets or sets the HTTP status code
+        /// </summary>
+        public HttpStatusCode StatusCode { get; set; }

```

Comment:
@horsdal, what are your thoughts on including the HTTP status code in the error handler result? Technically this is duplicated in the HTTP response status.